packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -47,6 +47,16 @@ abstract class ScrollActivityDelegate {
   /// Terminate the current activity and start a ballistic activity with the
   /// given velocity.
   void goBallistic(double velocity);
+
+  /// Some ballistic activities may want to allow input on child widgets.
+  /// This flag takes precedence over [ScrollActivity.shouldIgnorePointer] when
+  /// starting a ballistic activity.
+  bool? get preferredBallisticIgnorePointer;
```

Comment:
What does it mean when this is null? It looks like some scroll positions have an opinion for this and some don't. Is this only relevant with PageView? If so, why?